analyzers/tests/SonarAnalyzer.UnitTest/Common/MetricsTest.cs

```diff
@@ -189,306 +189,327 @@ public void CommentsWithHeaders()
         [TestMethod]
         public void Classes()
         {
-            Classes(AnalyzerLanguage.CSharp, "").Should().Be(0);
-            Classes(AnalyzerLanguage.CSharp, "class MyClass {}").Should().Be(1);
+            Classes(AnalyzerLanguage.CSharp, string.Empty).Should().Be(0);
+            Classes(AnalyzerLanguage.CSharp, "class Sample {}").Should().Be(1);
```

Comment:
[curiosity, not actionable]  why was this change needed?
if it was to simplify, then why
```
IMyInterface
MyEnum
MyRecord
```
haven't been updated, like in `Accessors()`?

Reply:
VB.NET has a `MyClass` keyword that would require VB and C# naming to be different (more painful).

I also find `class MyClass` more difficult to read. `class Sample` clearly distinguishes between the structure (keyword) and data.